nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/jquery/propertytable/jquery.propertytable.js

```diff
@@ -2115,32 +2143,36 @@
                                         var row = propertyData.getRowById(id);
                                         propertyGrid.setActiveCell(row, propertyGrid.getColumnIndex('value'));
                                         propertyGrid.editActiveCell();
-                                    });
-                                } else {
-                                    // if this row is currently hidden, clear the value and show it
-                                    if (existingItem.hidden === true) {
-                                        propertyData.updateItem(existingItem.id, $.extend(existingItem, {
-                                            hidden: false,
-                                            previousValue: null,
-                                            value: null
-                                        }));
-
-                                        // select the new properties row
-                                        var row = propertyData.getRowById(existingItem.id);
-                                        propertyGrid.setActiveCell(row, propertyGrid.getColumnIndex('value'));
-                                        propertyGrid.editActiveCell();
                                     } else {
-                                        nfDialog.showOkDialog({
-                                            headerText: 'Property Exists',
-                                            dialogContent: 'A property with this name already exists.'
-                                        });
-
-                                        // select the existing properties row
-                                        var row = propertyData.getRowById(existingItem.id);
-                                        propertyGrid.setSelectedRows([row]);
-                                        propertyGrid.scrollRowIntoView(row);
+                                        // if this row is currently hidden, clear the value and show it
+                                        if (existingItem.hidden === true) {
+                                            propertyData.updateItem(existingItem.id, $.extend(existingItem, {
+                                                hidden: false,
+                                                previousValue: null,
+                                                value: null
+                                            }));
+
+                                            // select the new properties row
+                                            var row = propertyData.getRowById(existingItem.id);
+                                            propertyGrid.setActiveCell(row, propertyGrid.getColumnIndex('value'));
+                                            propertyGrid.editActiveCell();
+                                            propertyGrid.invalidateRow(row);
+                                            propertyGrid.render();
```

Comment:
I believe calls to `invalidateRow` and `render` are preventing the `editActiveCell` from opening the editor for this row.

Reply:
I assume the call to `invalidateRow` and `render` might have been needed for the updated descriptor to take effect. I didn't try but maybe we can reorder the calls here.

Reply:
Thanks for catching that issue, invalidating the row is necessary for the value formatter to evaluate the updated sensitive status, reordering the calls works as expected.